plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/AppEngineConfigFileSchemaLocationProvider.java

```diff
@@ -0,0 +1,49 @@
+
+package com.google.cloud.tools.eclipse.appengine.validation;
+
+import java.net.URI;
+import java.util.Collections;
+import java.util.Map;
+import org.eclipse.core.runtime.Path;
+import org.eclipse.wst.xml.core.contentmodel.modelquery.IExternalSchemaLocationProvider;
+
+/**
+ * Associate schemas for App Engine standard environment configuration files.
+ * 
+ * @seealso the {@code org.eclipse.wst.xml.core.externalSchemaLocations} extension point
```

Comment:
Ah, I guess it should always be `@see <some.class.type>`. You could probably do `@see org.eclipse.wst.xml.core.externalSchemaLocations for extension point`? Not sure if it will format correctly.